tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -2126,11 +2126,18 @@ TEST_F(OpConverterTest, ConvertActivation) {
         "Relu is only implemented for tensors, at my_act");
   }
 
+  const float ALPHA = 0.2f;
+
   // Get nodedef for activation layer.
-  auto get_act_nodedef = [](string op_name) -> NodeDef {
+  auto get_act_nodedef = [ALPHA](string op_name) -> NodeDef {
```

Comment:
I believe that we don't need to capture constexpr or primitive const type variables.
See https://stackoverflow.com/a/49027502/9501553